assembl/static2/js/app/components/common/flatList.jsx

```diff
@@ -100,7 +106,13 @@ class FlatList extends React.Component<FlatListProps> {
   };
 
   render() {
-    const { networkStatus, items, ListItem, itemData, className } = this.props;
+    const { hash } = window.location;
```

Comment:
can we avoid to use `window.location` by using `react-router` (location prop) ?

Reply:
ok, I now use withRouter